bigbluebutton-html5/imports/ui/components/video-preview/component.jsx

```diff
@@ -400,6 +406,8 @@ class VideoPreview extends Component {
       });
       this.video.srcObject = stream;
       this.deviceStream = stream;
+
+      if (skipVideoPreview) setTimeout(this.handleStartSharing, skipVideoPreviewTimer);
```

Comment:
I think the handleStartSharing call should be over here, https://github.com/bigbluebutton/bigbluebutton/pull/8319/files#diff-68f9bb887cedc53137f14f6ccd8d76d4R247. If it's put there then it will skip all of the unnecessary parts that display the preview video.